public-docs/community-resources.md

```diff
@@ -0,0 +1,41 @@
+---
+id: community-resources
+title: Community Resources
+---
+
+This page is here to guide you through the Reaction ecosystem. As we've stated before, we wouldn't be here without our very engaged community. Here we'd like to introduce you to some of the best things we've seen out there.
+
+_**Note:** If you miss something that's not listed here and want to see it here, you're invited to open a [pull request](https://github.com/reactioncommerce/reaction-docs/pull/new/trunk)._
+
+## Projects maintained by Reaction
```

Comment:
Should Projects maintained by Reaction be in a separate file?

Reply:
One can argue that these projects are also community projects to an extent... They're not part of the core, and they're maintained by both the Reaction core team and the community. I'd say it would be good to keep them here, it'll avoid confusion for newcomers. Maybe when the list gets really long we can split it in two pages, but I don't think it's necessary for now. Also, for what it's worth, that's the way it was done in the previous Community Resources page.